test/fixtures/README.md

```diff
@@ -3,14 +3,14 @@
 We should use fixtures stored by entity to compose endpoints.
 For most cases the support API mutators should be used to test particular data.
 Other than what's in `test/` no fixture data shoud be used in tests.
-Ideally the fixture data (other than tests) can be randomly generated in the future.
 
 The files in the root of `fixtures/` are singular resources that are not requested
 as collections within the app.
 
 ### `collections/`
 
-Collections include arrays of multiple entities.
+Collections include arrays of multiple entities. Collections are automatically
+generated using faker when running tests.
```

Comment:
could probably use better documentation...this was the first thing I did last week or whenever I started on this

Reply:
Yeah so we should document
```
### `config/`
What this folder includes and what to do about it
```